jetty-core/jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -1078,7 +1078,7 @@ public StartEnvironment parse(StartEnvironment environment, String arg, String s
             return environment;
         }
 
-        if (arg.startsWith("--include-jetty-dir="))
+        if (arg.startsWith("--include-jetty-dir=") || arg.startsWith("--add-base-dir="))
```

Comment:
"add-base-dir" does not convey the priority, which was the objection raised for "parent-base-dir".

"default-base-dir" is not optimal because it seems to hint that you can avoid specifying a $JETTY_BASE because there is some default base dir.

I think I like better "common-base-dir".
Thoughts?

Reply:
> "add-base-dir" does not convey the priority, which was the objection raised for "parent-base-dir".

Greg didn't like parent -> https://github.com/jetty/jetty.project/issues/5946#issuecomment-773467925
I'm not a fan either, generally because I don't want to see words that have hierarchy or order as options.

> "default-base-dir" is not optimal because it seems to hint that you can avoid specifying a $JETTY_BASE because there is some default base dir.

Nope, this isn't a great name either, as you have discovered.

> I think I like better "common-base-dir".

Nah, this doesn't work either, as it implies an order.

Consider this setup ...

* the $jetty.base - has two entries `--include-jetty-dir=/opt/sitefoo/` and `--include-jetty-dir=/opt/audit/`
* the `/opt/sitefoo/` has an entry `--include-jetty-dir=/opt/sitecorporate/`

That makes `/opt/sitefoo/` and `/opt/audit/` at the same level (before `jetty.base`), but `/opt/sitecorporate/` (before `/opt/sitefoo/`)

None of these are "base dirs".
These are just jetty configuration sources.
Hence the original use of `--include-jetty-dir=<dir>` as a name.

Note: keep in mind that even the `$jetty.home` directory can have configuration like a `start.d/*.ini` file too.

Reply:
@joakime so perhaps "add-include-dir"?

Can we have an "include" dir, and a totally empty "base" dir?

I mean this:
```
mkdir base
cd base
java -jar start.jar --include-dir=/opt/corp/jetty
```

Did not try, but I think it should start Jetty fine, provided that `/opt/corp/jetty` has a minimal config.